include/deal.II/grid/construction_util.h

```diff
@@ -0,0 +1,120 @@
+// ---------------------------------------------------------------------
```

Comment:
I'd say just merge this file into the one where you declare the relevant data structures.